snippets/csharp/programming-guide/strings/composite_1.cs

```diff
@@ -0,0 +1,16 @@
+// Composite Formatting
```

Comment:
This file is not necessary, as the file with the program already contains this code.

Reply:
Removed.